File: baekjoon/bronze/level1/test_2869/java/Main.java
package bronze.level1.test_2869.java;

import java.io.*;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int[] input = Stream.of(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int count = (input[2] - input[0]) / (input[0] - input[1]);
        count += (input[2] - input[0]) % (input[0] - input[1]) > 0 ? 1 : 0;

        bw.write(String.valueOf(count + 1));
        bw.close();
    }
}

File: baekjoon/bronze/level1/test_2869/kotlin/Main.kt
package bronze.level1.test_2869.kotlin

import kotlin.math.ceil

fun main() {
    val input = readln().split(" ").map { it.toDouble() }
    val a = input[0]
    val b = input[1]
    val v = input[2]

    val answer = ceil((v - a) / (a - b)) + 1
    print(answer.toInt())
}
